static/app/views/performance/browser/webVitals/webVitalsLandingPage.tsx

```diff
@@ -174,6 +175,32 @@ export function WebVitalsLandingPage() {
                 />
               </WebVitalMetersContainer>
               <PagePerformanceTable />
+              <PagesTooltipContainer>
+                <Tooltip
+                  title={
+                    <div>
+                      <div>
+                        {tct(
+                          'If your instrumented pages are not appearing in the table below, you may be missing necessary web vitals in your page loads due to unsupported browsers or missing instrumentation. Find out what browsers are supported here [link:here].',
```

Comment:
```suggestion
                          'If pages you expect to see are missing from the table below, your framework is most likely not supported by the SDK, or your traffic is coming from unsupported browsers. Find out the supported browsers and frameworks here [link:here].',
```

Reply:
I might even shorten this further to:

If pages you expect to see are missing, your framework is most likely not supported by the SDK, or your traffic is coming from unsupported browsers. Find supported browsers and frameworks here [link:here].